.gitignore

```diff
@@ -62,3 +62,4 @@ Temporary Items
 .apdisk
 *.csv#
 Maps/TMX2EA Readme.txt
+VBA-BT_01.png
```

Comment:
Might as well just make this `*.png` for all screenshots taken in the default directory.